apps/design-land/src/scss/styles.scss

```diff
@@ -3,26 +3,27 @@
 ************************************/
 
 @use 'global';
+@use "sass:meta";
```

Comment:
I don't think stylelint is enabled for design-land, but all the double quotes in the file should be single quoates.